reorder_python_imports/main.py

```diff
@@ -245,11 +245,22 @@ def apply_import_sorting(partitions, **sort_kwargs):
     )
 
     new_imports = []
+    relative_imports = []
+
+    separate_relative = sort_kwargs.pop('separate_relative', False)
```

Comment:
since this isn't part of the kwargs passed onto `sort`, I think it should be an explicit parameter in this function?  though I guess this works, feel free to drop